tests/e2e/integration/auth.js

```diff
@@ -5,15 +5,13 @@ describe('Test cases for login functionality', () => {
       .should('be.visible')
 
       .shouldRedirect('/', '/account')
-      .shouldRedirect('/import-account', '/account')
-      .shouldRedirect('/more/about/terms', '/more/about/terms')
+      .shouldRedirect('#/more/about/terms', '/more/about/terms')
```

Comment:
Why have we returned to the `#` infront of the the route? Are the tests run on the web?

Reply:
`vue/router` in develop is initialized with `mode: IS_WEB ? 'history' : 'hash',`
after migrating to Vue3 it is initialized with `history: IS_WEB ? createWebHistory() : createWebHashHistory(),`
So if `IS_WEB` is not set we are using hash history. When running tests we are not specifying `IS_WEB` but if we do then the `#` is not required anymore

Reply:
Should I add `PLATFORM=web` when running e2e tests and remove the `#`?